recipes/python-neovim/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "neovim" %}
+{% set version = "0.1.13" %}
+{% set sha256 = "fd41cef3caa34a8b685d2c0603022fb352e5c2f07c666594215d4a998629f35f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - msgpack-python >=0.4.0
+    - greenlet
+    - trollius  # [py27]
+
+test:
+  imports:
+    - neovim
+    - neovim.api
+    - neovim.msgpack_rpc
+    - neovim.msgpack_rpc.event_loop
+    - neovim.plugin
+
+about:
+  home: http://github.com/neovim/python-client
+  license: Apache-2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Looks like the `license_file` section can be simplified.  Made the edit within 43bc154 all built successfully.